bm_scrapy/producer.py

```diff
@@ -18,15 +18,15 @@ def connect_kafka_producer():
         _producer = KafkaProducer(
             bootstrap_servers=bootstrap_servers,
             value_serializer=lambda x: json.dumps(x).encode("utf-8"),
+            api_version=(0, 10),
```

Comment:
Why are we adding api version?

Reply:
It turns out that we actually need this field 😅